ocropus-nlbin

```diff
@@ -192,7 +192,7 @@ def process1(job):
     bin = 1*(flat>args.threshold)
 
     # output the normalized grayscale and the thresholded images
-    print_info(fname+" lo-hi (%.2f %.2f) angle %4.1f"%(lo,hi,angle) + comment)
+    print_info("%s lo-hi (%.2f %.2f) angle %4.1f %s" % (fname, lo, hi, angle, comment))
```

Comment:
The first two spaces are because of `%4.f` for a value < 10. The second two spaces are fixed now.